src/defines_bmi160.h

```diff
@@ -0,0 +1,73 @@
+/*
+    SlimeVR Code is placed under the MIT license
+    Copyright (c) 2021 Eiren Rain
```

Comment:
Copyright here and in other new files should be `Copyright (c) 2022 SlimeVR Contributors`